changelog/pending/20241106--sdk-go--workspace-copytemplatefiles-overwrites-directories-when-called-with-force-true.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: fix
+  scope: sdk/go
+  description: workspace.CopyTemplateFiles overwrites directories when called with force=true (#17653)
```

```suggestion
  description: Overwrite directories in workspace.CopyTemplateFiles when called with force=true
```

The issue number is added automatically to the changelog by our tooling.  Also the changelog should be written in the active imperative form, see https://github.com/pulumi/pulumi/blob/master/CONTRIBUTING.md#changelog-messages.